addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -5395,6 +5395,10 @@ registry.ReplaceMedia = SnippetOptionWidget.extend({
      * @see this.selectClass for parameters
      */
     async replaceMedia() {
+        // ​Inspired from Wysiwyg.startEdition: this.$editable.on('dblclick'
+        // Fixes previous issue of openMediaDialog with sel.rangeCount === 0
+        // not sure it's OK here
+        $(this.$target[0]).selectElement();
```

Comment:
Not having this prevents re-editing icons with the "Replace Media" button. Specifically, the call reaches `openMediaDialog` with a state such that `sel` (a variable containing the `document.getSelection()` has `rangeCount === 0`

Reply:
This is no longer the case in my latest round of testing so no idea how to justify it anymore. Will wait until issue with s_company_team icons is fixed so I can test it there too.

Reply:
It is once more needed for the replace button to work on images/icons on the _first_ attempt to openMediaDialog.
Although, if you manage to open the media dialog in any other way (double-click, replace other editable content) then it works once more.
Not added back in case you want this in another task @dmo-odoo